### DISCUSSION TITLE
Problem with columns on iPhone Mail

### DISCUSSION THREAD

**Discussion by macmastermike** [bootstrap-email/bootstrap-email#168] [Help] [answered]
Hi,
i have an issue with the row/columns on apple mail.
somehow the media querys gets ignored (or removed?) so a 2/3/.. column layout always stays as column layout.
seems the 600px media query with the 100% width and display:block is "missing"...

does someone else have this issue? everything works fine on android devices and outlook mail mobile

**Comment by stuyam** [comment]
Could you share a code example and screenshot of what you are experiencing so we can take a look?

**Reply by stuyam** [reply]
Also what device are you using? Certain larger iPhones are larger than 600px wide and therefor won't drop into the media query.

**Reply by matthewrk** [reply]
I'm also seeing this on iPhone 11 Pro (not max version). A simple grid with col-lg-4 columns will show it. It looks as expected with the columns stacking vertically on mobile in the online editor as well as in Gmail's iOS app, but they're still horizontal on Apple Mail.

**Reply by matthewrk** [reply]
Interesting observation, it works correctly on 0.3.4.

**Reply by matthewrk** [reply]
I've found the problem to be the DOCTYPE tag. Changing it back to XHTML from HTML 4 Transitional restores the responsiveness again in Apple Mail. Apparently under the latter Apple Mail will not honour the display:block; on a TD tag.

**Comment by stuyam** [comment, marked as answer]
Turns out this was an issue with the online editor picking a random doctype if there wasn't one in the document. Now it will replace the doctype with the correct XHTML one. Thanks @matthewrk for pointing me in the right direction here thank you! https://github.com/bootstrap-email/bootstrap-email/releases/tag/1.1.4
